03/app.js

```diff
@@ -42,23 +47,18 @@ const createListItem = (file) => {
     const newListLitem = listItemPrototype.cloneNode(true);
     newListLitem.classList.remove(listItemPrototypeClass);
 
-    const nameElem = newListLitem.querySelector(listItemNameSelector);
-
-    if (nameElem) {
-      nameElem.innerText = file.name;
-    }
-
-    const imageElem = newListLitem.querySelector(listItemImageSelector);
-
-    if (imageElem) {
-      renderFile(file, imageElem);
-    }
-
-    const sizeElem = newListLitem.querySelector(listItemSizeSelector);
-
-    if (sizeElem) {
-      sizeElem.innerText = convertSizeToMB(file.size);
-    }
+    updateListItemElementContent(newListLitem, listItemNameSelector, file.name);
+    updateListItemElementContent(
+      newListLitem,
+      listItemSizeSelector,
+      convertSizeToMB(file.size)
+    );
+    updateListItemElementContent(
+      newListLitem,
+      listItemImageSelector,
+      renderFile,
+      file
```

Comment:
Można było zrobić coś na zasadzie `renderFile.bind(this, file)` - tworzysz kopie funkcji, która jest uruchomiona ze wskazanymi parametrami.